src/main/java/seedu/planner/model/ModelManager.java

```diff
@@ -132,53 +126,74 @@ public Module finalizeModule(Module module) {
         return finalizedModules;
     }
 
-    //@@author RomaRomama
-
     @Override
     public void addModules(Set<Module> modules, int index) {
         Set<Module> finalizedModules = finalizeModules(modules);
         versionedModulePlanner.addModules(finalizedModules, index);
         indicateModulePlannerChanged();
     }
 
-    //@@author Hilda-Ang
-
+    /**
```

Comment:
I don't think need javadocs for overridden methods since their base methods already have the javadocs

Reply:
Actually I'm not sure whether to put javadocs on the base methods, overriden methods, or both

Reply:
Address Book 4 puts javadocs for the base methods. At least from what I saw from `Model` and `ModelManager`. I saw got this `inheritDoc` annotation to inherit docs from the base method but I don't think we need that for this project. So javadocs for base methods bah

Reply:
Done